jobs/acceptance-tests-brain/templates/pre-start.erb

```diff
@@ -4,7 +4,9 @@ set -o errexit
 set -o nounset
 
 PATH=$PATH:/var/vcap/packages/cli/bin
+API="<%= properties.smoke_tests.api %>"
+PASSWORD="<%= properties.acceptance_tests_brain.password %>"
 
-cf api --skip-ssl-validation "https://api.$DOMAIN"
-cf auth admin "${CLUSTER_ADMIN_PASSWORD}"
+cf api --skip-ssl-validation "${API}"
```

Comment:
Looking at [`properties.smoke_tests.api`](https://github.com/SUSE/brain-tests-release/blob/fe15bb7fa6e9f681644dbb02517296ae9af0121e/jobs/acceptance-tests-brain/spec#L45) it doesn't contain the scheme (`https://`); should we not leave that part in?  Or use [`acceptance_tests_brain.domain`](https://github.com/SUSE/brain-tests-release/blob/fe15bb7fa6e9f681644dbb02517296ae9af0121e/jobs/acceptance-tests-brain/spec#L23) instead?

Also, I don't think we actually need temporary variables here:
```suggestion
cf api --skip-ssl-validation https://api.<%= acceptance_tests_brain.domain.shellescape %>
```

Yeah, we might need to do shell escape, especially for that password…

Reply:
In the brain test instance group I have

```
        smoke_tests:
          api:                 https://api.((system_domain))
```

... Hm. I made this different from the scf property. Ok. I will change the property in the manifest to match what we had in scf, i.e. without scheme, and then add it back in the release.

Reply:
Addressed with commit 39e2b8112e304c5d8b3310bb13cb94fce61c83e7.